pkg/resources/saml2_integration_acceptance_test.go

```diff
@@ -0,0 +1,292 @@
+package resources_test
+
```

Comment:
there are no tests for external changes behavior (with/without defaults), there are no tests with zero values. You can take something from database/warehouse.

Reply:
@sfc-gh-jcieslak

Reply:
missed that one somehow, I'll add some